### DISCUSSION TITLE
File upload overwrite

### DISCUSSION THREAD

**Discussion by TerryLiang404** [requarks/wiki#6013] [Error / Bug Report] [answered]
It is found that if a file with the same name is uploaded, the old file will be overwritten.
For example. there is a test.jpg in the system, if I upload another test.jpg file to the system, the new test.jpg will overwrite the old one.

**Comment by fnafox** [comment, marked as answer]
This is the intended behavior.

**Reply by TerryLiang404** [reply]
But there is no prompt when uploading, some files are overwritten and we do not know.
